src/main/java/org/terasology/launcher/github/GitHubRelease.java

```diff
@@ -46,4 +51,14 @@ public String getTagName() {
     public String getBody() {
         return json.getString("body");
     }
+
+    public String getDownloadUrl(String platform) {
```

Comment:
Instead of putting utility methods with logic in here, just expose `getAssets` which returns a `List<GithubAsset>` (introduce new class). That class will provide means to `getName()` and `getBrowserDownloadUrl()`.

Reply:
Easy to implement :+1:. That's surely the most object-oriented approach, but let me ask if it's really necessary? Like if we are supposed to work with a single target asset (depending on the user's platform), why should we provide a list of assets or their names? :slightly_smiling_face:

Reply:
I would like to keep all the classes in the `github` package as generic as possible, so that we could exchange them easily.

The current way we use the github api library is not ideal, as we are dealing with the plain JSON objects. In principle, those libraries offer interfaces similar to what is implemented in that package, so I'm preparing an easy switch.